docpages/example_programs/misc/setting_status.md

```diff
@@ -2,8 +2,6 @@
 
 A bot status is pretty cool, and it'd be cooler if you knew how to do it! This tutorial will cover how to set the bot status to say `Playing games!`, as well as covering how to set the status to the amount of guilds every two minutes.
 
-\note dpp::get_guild_cache requires the bot to have the guild cache enabled, if your bot has this disabled then you can't use that. Instead, you should look to use dpp::cluster::current_application_get and get the `approximate_guild_count` from dpp::application in the callback.
```

Comment:
why is this removed? it is a valid and important note.

Reply:
Because the code now uses that function, I made it no longer rely on the cache.

Reply:
True, I'll undo that part then.